core/src/cluster_info.rs

```diff
@@ -300,6 +300,7 @@ pub struct ClusterInfo {
     socket: UdpSocket,
     local_message_pending_push_queue: RwLock<Vec<(CrdsValue, u64)>>,
     contact_debug_interval: u64,
+    instance: RwLock<NodeInstance>,
```

Comment:
can `instance` just be the unique u64 value? Everything else can be constructed at send time, no? And not changing, so doesn't need a lock.

Reply:
We at least need:
* a timestamp of when the instance started, to stop the older one.
* and also, a randomly generated token, to distinguish the instances if both start at the same milli-second (e.g. programmatically)

Nevertheless, we do not need to update the wallclock here, so I can remove `RwLock`, and only update the wallclock on the value which is pushed through gossip.

Reply:
I think if `instance` was a UTC timestamp at ms or even second accuracy we'd get 99.9% of the way there.   If two instances happen to start at exactly the same time then they both exit.

Reply:
Current code no longer has `RwLock` so I guess should be good.

Alternative is to use `std::time::SystemTime::now()` as the timestamp of when the validator started (as opposed to milliseconds from `timestamp()`), and assume that no two validators can start at the exact same `SystemTime`. Then we no longer need the random `token`, and one single immutable `SystemTime` should suffice.

Reply:
wfm